packages/perseus/src/widgets/__tests__/passage_test.jsx

```diff
@@ -130,4 +167,48 @@ describe("passage widget", () => {
         // Assert
         expect(reference).toBe(null);
     });
+
+    it("should render passage title", () => {
+        renderPassage({passageTitle: "Hello World"});
+
+        expect(screen.getByText("Hello World")).toBeInTheDocument();
+    });
+
+    it("should render passage text", () => {
+        renderPassage({passageText: "Hello World"});
+
+        expect(screen.getByText("Hello World")).toBeInTheDocument();
+    });
+
+    it("should render footnotes", () => {
+        renderPassage({footnotes: "Hello World"});
+
+        expect(screen.getByText("Hello World")).toBeInTheDocument();
+    });
```

Comment:
Question (non-blocking): Should these each use a different phrase? Is there any chance the checks could be seeing "Hello World" from a previous render? Might also be more clear if they used different phrases related to what was being changed; maybe something like "This is passageText", etc. Probably doesn't have much impact since these are just for illustration, but just curious.

Reply:
It shouldn't bleed through like that, but it wouldn't hurt to change the text. Updated.